public/js/user-store.js

```diff
@@ -18,6 +18,10 @@ function UserStore(localDispatcher) {
   localDispatcher.register((function(payload) {
     if (payload.actionType === 'set-user') {
       console.log('UserStore: storing current user:', payload.user);
+      //
+      // Example:
+      // {email: 'f@f.com', is_logged_in: true, payment_methods: []}
```

Comment:
I like this. Is nice.

![article-2450794-05e6dc090000044d-375_634x403](https://cloud.githubusercontent.com/assets/23885/8258130/c49530ea-166f-11e5-92ff-0207aaf22efb.jpg)